internal/analyzer/aggregator_test.go

```
@@ -579,3 +579,4 @@ func TestNewAggregator(t *testing.T) {
 	aggregator := NewAggregator()
 	assert.NotNil(t, aggregator)
 }
+
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)